pkgs/development/tools/nimble/default.nix

```diff
@@ -1,37 +1,32 @@
-{ stdenv, fetchFromGitHub, nim }:
-
-let
-  srcs = {
-    nimble = fetchFromGitHub {
-      owner = "nim-lang";
-      repo = "nimble";
-      rev = "v0.7.2";
-      sha256 = "0j9b519cv91xwn6k0alynakh7grbq4m6yy5bdwdrqmc7lag35r0i";
-    };
-    nim = fetchFromGitHub {
-      owner = "nim-lang";
-      repo = "nim";
-      rev = "v0.13.0";
-      sha256 = "14grhkwdva4wmvihm1413ly86sf0qk96bd473pvsbgkp46cg8rii";
-    };
-  };
-in
+{ stdenv, fetchFromGitHub, nim, openssl, makeWrapper }:
+
 stdenv.mkDerivation rec {
   name = "nimble-${version}";
-  version = "0.7.2";
 
-  src = srcs.nimble;
+  version = "0.7.4";
+
+  src = fetchFromGitHub {
+    owner = "nim-lang";
+    repo = "nimble";
+    rev = "v${version}";
+    sha256 = "1l477f1zlqpc738jg47pz599cwjasgy9jqdsplj3ywd12xfqpc96";
+  };
 
-  buildInputs = [ nim ];
+  buildInputs = [ nim makeWrapper ];
 
-  postUnpack = ''
-    mkdir -p $sourceRoot/vendor
-    ln -s ${srcs.nim} $sourceRoot/vendor/nim
+  patchPhase = ''
+    substituteInPlace src/nimble.nim.cfg --replace "./vendor/nim" "${nim}/share"
   '';
-  buildPhase   = ''
+
+  buildPhase = ''
     nim c src/nimble
   '';
-  installPhase = "installBin src/nimble";
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp src/nimble $out/bin
+    wrapProgram $out/bin/nimble --prefix LD_LIBRARY_PATH : "${openssl.out}/lib"
```

Comment:
@joachifm Sorry for my vague "did not work" response. I ran nimble and it failed to load libcrypto.so (don't have the exact message right now). 
I need to call it a day for now (as 2AM is approaching), but will try to sort this out, and the hash issue, tomorrow.